src/components/MovieDetails.js

```diff
@@ -6,24 +6,45 @@ class MovieDetails extends Component {
     super(props)
     this.state = {
       id: props.location.state.movie.id,
-      movie: {}
+      movie: {},
+      isLoading: true,
+      error: null
     }
   }
 
   componentDidMount = () => {
     fetch(`https://rancid-tomatillos.herokuapp.com/api/v2/movies/${this.state.id}`)
```

Comment:
It'll probably be helpful to break this (and other network request functions) into its own apiCalls.js file eventually